src/_guides/language/effective-dart/style.md

```diff
@@ -98,11 +98,10 @@ extension SmartIterable<T> on Iterable<T> { ... }
 
 [extensions]: /guides/language/extension-methods
 
-### DO name libraries, packages, directories, and source files using `lowercase_with_underscores`. {#do-name-libraries-and-source-files-using-lowercase_with_underscores}
+### DO name packages, directories, and source files using `lowercase_with_underscores`. {#do-name-libraries-and-source-files-using-lowercase_with_underscores}
 
-{% include linter-rule-mention.md rule1="library_names" rule2="file_names" %}
+{% include linter-rule-mention.md rule="file_names" %}
```

Comment:
I guess we may have missed this before, but there is a `package_names` rule now we could mention: https://dart.dev/tools/linter-rules#package_names

```suggestion
{% include linter-rule-mention.md rule1="file_names" rule2="package_names" %}
```